collections/guides/quickstarts/quickstart_java.md

```diff
@@ -0,0 +1,234 @@
+
+<img src="/assets/images/getting_started/application_permissions.png" />
+
+### 3.4 Copy your API key
+Keep this API key safe for later. We will use it in the next step to create your first Token.
+
+<img src="/assets/images/getting_started/application_api_key.png" />
+
+## Step 4: Create a Token to secure a string
+To create a token, we need to send an HTTP POST request to the [`/tokens`](https://docs.basistheory.com/api-reference/#tokens-create-token) endpoint. In this guide, we will be using the `token` Token Type (you can read more about Tokens [here](https://developers.basistheory.com/concepts/what-are-tokens/)).
+
+Update the `BT-API-KEY` header with the API Key you created in Step 3!
+
+(Note: some boilerplate code is missing in this snippet. The complete class is at the bottom of this guide!)
+
+```java
+// Initialize the HttpClient
+var client = HttpClient.newHttpClient();
+
+// JSON message formatted as a string
+var message = "{\"type\": \"token\", \"data\": \"foo\"}";
+
+var createRequest = HttpRequest.newBuilder()
+    .uri(URI.create("https://api.basistheory.com/tokens"))
+    .POST(BodyPublishers.ofString(message))
+    .header("Content-type", "application/json")
+    .header("Accept", "application/json")
+    // Enter your API key here!
+    .header("BT-API-KEY", "<! ENTER YOUR API KEY HERE !>")
+    .build();
+
+// Create a new token
+var createResponse = client.send(createRequest, BodyHandlers.ofString());
+var token = createResponse.body();
+
+// Token response
+System.out.println("Create a token:");
+System.out.println(token);
+```
+
+## Step 5: Run your application to create a new Token
+To create a token, run the following commands in the directory you created your `BasisTheoryQuickstart.java` file:
+
+```bash
+javac BasisTheoryQuickstart.java
+java BasisTheoryQuickstart
+```
+
+You will see a response similar to:
+```bash
+> Create a token:
+{"id":"62a71684-f148-424c-88c6-bdb44031357a","tenant_id":"570b53fb-1ecf-4aaf-9cb2-145e13b566a9","type":"token","privacy":{"classification":"general","impact_level":"high","restriction_policy":"redact"},"created_by":"7281f0ef-eafc-455c-bdae-ce6c99ff8268","created_at":"2022-02-17T21:49:29.2596915+00:00"}
+```
+
+🎉🎉🎉 You’ve created a token 🎉🎉🎉
+
+## Step 6: Read back the raw value from Basis Theory
+
+With our value safely stored in a Token, let’s read that value back to our system. To do this, we will make an HTTP GET request to the [Basis Theory Get a Token API](https://docs.basistheory.com/api-reference/#tokens-get-a-token) endpoint and print the response's raw string value.
+
+We are using the `id` property from the previous Token we created to inject the `tokenId` into the Get a token request. Because we are using vanilla Java, we will simply split the response and grab the `id` from the resulting array.
```

Comment:
```suggestion
We are using the `id` property from the previous Token we created to inject the `tokenId` into the Get token by id request. Because we are using vanilla Java, we will simply split the response and grab the `id` from the resulting array.
```